algorithms/unit_tests/TestRandom.hpp

```diff
@@ -327,10 +327,6 @@ template <class RandomGenerator, class Scalar>
 struct test_random_scalar {
   using rnd_type = typename RandomGenerator::generator_type;
 
-  int pass_mean, pass_var, pass_covar;
-  int pass_hist1d_mean, pass_hist1d_var, pass_hist1d_covar;
-  int pass_hist3d_mean, pass_hist3d_var, pass_hist3d_covar;
-
```

Comment:
These changes look good but why did you push them here?

Reply:
I was trying to debug the issue and wanted better error messages. I thought it was reasonable to keep these changes but I can move them somewhere else if you prefer.

Reply:
It's ok by me.